### ISSUE TITLE
Scan Successful Screen improvements

### ISSUE THREAD

**Issue by eriksogn** [zerobase-io/smart-tracing#196]
#### Overall
Modal dialog stretched to 100% height
Blurred background removed

#### Check-in Successful
I changed this from "Scan Successful" because "check in successful" is more impactful and important to our individual than the mechanism of scanning.

#### Thank you
If the check in location has a region: "Thank you for protecting Region"
Otherwise: "Thank you for protecting our community"
The heart gives us humanity.

#### Scan result
If the location has a title: "Trader Mo’s 7th st at 6:32 PM" 
Otherwise "Successful Scan at 6:32 PM"
The purpose of the title and time on scan is to deter forgery.
The barcode is removed to declutter, simplify, and because it is not used by anyone currently.
I introduce the language of "touchpoint" here in a way that is understandable

#### Keep me notified
Pops up a modal dialog to re-iterate our privacy commitment, that we only collect this number for notifications.
"Keep me notified" saves the phone number
"I do not want to be notified" turns off notifications and forgets the phone number

#### Feedback
The entire purpose of our rollout is to validate our mechanism, so feedback gets a strong call to action.
Pops up a modal dialog with our simple feedback form

#### Learn more
I changed this from "Continue to Zerobase", as that can feel like there are additional steps to take once the user continues.

#### Quote
I added an inspirational quote here that can change with every successful scan.
Every check in is an opportunity to show them our heart and share an empowering message.

#### Footer
Added our tagline
Added footer links for ease of information and compliance

## Revision Mockups
![scan-screen-mockup](https://user-images.githubusercontent.com/5020291/78419073-b5292a80-75f6-11ea-8dc2-f92d21c35866.png)
![scan-screen-mockup-notification-settings](https://user-images.githubusercontent.com/5020291/78419070-b2c6d080-75f6-11ea-9e02-ca2ff591b6b6.png)

**Comment by JohnCLo** [comment]
Looks good, however will need to coordinate with backend if it makes sense to return "Manchester" "Trader Mo's" "7th Street", as this will require an additional query.

Additionally is it generally good practice to let users know the specific locations they are checking into? Or would it seem like a privacy burden and make people feel vulnerable? Could very well be "Manchester" "Gay Bar" "8th Street". How can we reconcile this.

Please also provide a list of rotating quotations for FE to upload.

**Comment by JohnCLo** [comment]
Looping in @joshhopkins @ColmByrne @miketitchen for peer review and visibility since this is a larger feature

**Comment by joshhopkins** [comment]
Looks good @eriksogn. I think for now not displaying the location name is best, it does feel like the user is being tracked – even if no personal ID is attached.

**Comment by miketitchen** [comment]
I agree, looks good but for now let's not display business name or location as that would require a change to contract with backend right now.

**Comment by eriksogn** [comment]
I have split three parts of this issue into three separate tickets: See #144 for the feedback feature, #243 for keep me notified, and #244 for the quote. The most critical issue here is the feedback mechanism.
